frontend/website/jobs/frontend-engineer.markdown

```diff
@@ -1,32 +1,51 @@
 ---
-title: Senior Frontend Engineer
-date: 2019-02-15
+title: Product Engineer (Frontend)
+date: 2019-06-20
 ---
-O(1) Labs is aiming to develop the first cryptocurrency protocol that can deliver on the promise of supporting real-world applications and widespread use. Our team is based in San Francisco, and we are funded by top investors (including Polychain, Metastable, Max Levchin, and Naval Ravikant).
 
-We're bringing Coda Protocol to market, a cryptocurrency that compresses the blockchain from hundreds of gigabytes down to the size of a few tweets. It can scale to thousands of transactions per second and millions of users while remaining decentralized enough for cellphones to be fully verifying nodes.
+Help us build accessible, usable, delightful products on top of the Coda Cryptocurrency Protocol!
 
-We’re working on technologies with the potential to reimagine social structures. We believe it’s important to incorporate diverse perspectives from conception through realization.
+We’re seeking a frontend product engineer who is also excited by building beautiful products. While it's hard to say exactly what we'll work on given that we are such an early-stage startup, we expect projects to include:
+
+- Wallet software that is a joy to use, with innovative onboarding and account recovery mechanisms.
+- The product side of developer tools and SDKs
+- Imaginative cryptocurrency applications that showcase what’s possible with a lightweight blockchain
+
+
+# Requirements
```

Comment:
nit: header feels too big relative to job title - maybe shift down one header size